downloadPdfCustomSelectionToolbar/downloadPdfCustomSelectionToolbar.js

```diff
@@ -46,11 +46,17 @@ var updateDownloadPdfCustomSelectionToolbarSelectButtonLabel = function(event) {
 };
 
 var updateDownloadPdfCustomSelectionIndicators = function (event) {
-    if (event.data.leftPageNumber == null || event.data.rightPageNumber == null || event.data.leftPageNumber === 0) {
+    if (event.data.leftPageNumber == null || event.data.leftPageNumber === 0) {
         downloadPdfCustomSelectionCheckBoxes.left.css('visibility', 'hidden');
     } else {
         downloadPdfCustomSelectionCheckBoxes.left.css('visibility', 'visible');
     }
+
+    if (event.data.rightPageNumber == null) {
```

Comment:
... but this is also true, as the first page is considered to be the left one (even though it's actually centred), so the right checkbox is also hidden.